### ISSUE TITLE
Fraction formatting - too little space

### ISSUE THREAD

**Issue by willymcallister** [Khan/KaTeX#1712]
The formatting of this compound fraction is awkward. Both the numerator and denominator are too close to the main fraction line.

$\dfrac{1}{Rb}=\dfrac{\dfrac{1}{R1\,R3}}{\dfrac{R2\,R3}{R1\,R2\,R3}+\dfrac{R1\,R3}{R1\,R2\,R3}+\dfrac{R1\,R2}{R1\,R2\,R3}}$

screenshot:
<img width="470" alt="screen shot 2018-09-12 at 10 36 41 am" src="https://user-images.githubusercontent.com/17035787/45442982-5a2c8d00-b678-11e8-81db-13aa3089d79d.png">

I'm using this version of KaTeX: 
src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.js"

This comes from the 5th equation from the end of https://spinningnumbers.org/a/delta-wye-derivations.html

**Comment by willymcallister** [comment]
You are right about the source code I put in my original message. The comma's should all be \,
(aside: That line of source code came was extracted by the copy-tex contributed extension. The back slashes didn't make it into the pasted text.)

The code is as you suggested:
$\dfrac{1}{Rb}=\dfrac{\dfrac{1}{R1\,R3}}{\dfrac{R2\,R3}{R1\,R2\,R3}+\dfrac{R1\,R3}{R1\,R2\,R3}+\dfrac{R1\,R2}{R1\,R2\,R3}}$

My point is the tight spacing where the minor fractions crowd the major fraction line (example: the space below R1 R3 in the numerator). The major fraction has very little spacing above and below the major fractionline, in contrast to the minor fractions that have significantly more space.

In the minor fractions the spacing to the fraction line is larger, but not symmetric. The numerators float higher above their short fraction lines than the denominators below the fraction line. 

When the major fraction is composed, the top and bottom are mashed close to the major fraction line.

I'm glad KaTex matches LaTeX. Since that is always a key objective then this observation is perhaps more about LaTeX and should be put aside.

**Comment by kevinbarabash** [comment]
Unfortunately this is an issue that affects LaTeX as well as evidenced by many questions on stack exchange.  TBH, I feel like TeX gets this wrong.  My expectation of nested fractions is that the child fractions should have their numerators and denominators closer to their fraction bars and the parent's numerators and denominators should be further away.
